app/src/main/java/com/kadequart/android/quartrack/MainActivity.java

```diff
@@ -41,24 +51,42 @@ public void onClick(View view) {
             }
         });
 
-//        RecyclerView recyclerView = (RecyclerView) findViewById(R.id.transactions_recycler_view);
-//        recyclerView.setHasFixedSize(true);
-//
-//        RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(this);
-//        recyclerView.setLayoutManager(layoutManager);
-//
-//        ArrayList<Transaction> transactions = new ArrayList<>();
-
-//        transactions.add(new Transaction("#00000325", 9842.99));
-//        transactions.add(new Transaction("#00000325", 9842.99));
-//        transactions.add(new Transaction("#00000324", 9209.24));
-//        transactions.add(new Transaction("#00000323", 9389.44));
-//        transactions.add(new Transaction("#00000322", 5523.23));
-
-//        RecyclerView.Adapter adapter = new TransactionAdapter(transactions);
-//        recyclerView.setAdapter(adapter);
-//
-//        DividerItemDecoration divider = new DividerItemDecoration(recyclerView.getContext(), DividerItemDecoration.VERTICAL);
-//        recyclerView.addItemDecoration(divider);
+        recyclerView = (RecyclerView) findViewById(R.id.transactions_recycler_view);
+        recyclerView.setHasFixedSize(true);
+
+        RecyclerView.LayoutManager layoutManager = new LinearLayoutManager(this);
+        recyclerView.setLayoutManager(layoutManager);
```

Comment:
`recyclerView.setLayoutManager(new LinearLayoutManager(this));` In-ani lang ta part kung dili na kailangan i reuse ang isa ka instance. Pero pag i reuse siya masking 2 times lang, assign nato sa variable.